### ISSUE TITLE
Support ticket - Jan 25 - Feb 7

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#4440]

**Comment by viviancan** [comment]
- [ ] Request from ChA team - requesting data pull for which ChA are most active - Completed on 1/31/24

**Comment by dboyer** [comment]
- [ ] Mentor completing training but training not marked as completed. The mentor, Kiki, and I all tried completing the training and received a message it was complete but no check mark in the side nav that let them continue with onboarding.

<img width="1188" alt="Screen Shot 2024-02-01 at 12 05 43 PM" src="https://github.com/Iridescent-CM/technovation-app/assets/516764/2cba75c4-0bf5-40c9-9c0c-8276c99c1923">

**Comment by viviancan** [comment]
Confirming the mentor training issue above has been resolved. I belive it was related to a few missing requirements in the mentor profile. More details [in this thread](https://iridescentlearning.slack.com/archives/C06CHBQ4E/p1706806232558129)

**Comment by viviancan** [comment]
When QA-ing background checks for ChA on preview, I ran into an error `ERROR: permission denied for table jobs` - After reviewing the db settings for the preview DB, I saw this error `ROWS WRITE ACCESS REVOKED`. This led me to a [help article from heroku ](https://help.heroku.com/JIFZYOSR/write-access-revoked-on-postgres-hobby-mini-basic-databases) that states upgrading to a paid plan will resolve this issue. I think we will need to upgrade the preview DB. 

 <img width="768" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMGdoQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--484fa65d0bb0a0f275019673ad20a768e3f7e938/CleanShot%202024-02-01%20at%2021.06.51%402x.png" alt="CleanShot 2024-02-01 at 21.06.51@2x.png" />
